src/grid/index.js

```diff
@@ -12,11 +12,13 @@ function Grid({
   style,
   children,
   componentClass: ComponentClass,
+  noPadding,
 }) {
   const className = cn(
     {
       [classes.grid]: !fluid,
       [classes.gridFluid]: fluid,
+      [classes['noPadding']]: noPadding,
```

Comment:
dot notation doesnt fit?

Reply:
I just copied how it is on the column. I can change both to use dot notation.